PSKoans/Koans/Constructs and Patterns/AboutErrorHandling.Koans.ps1

```diff
@@ -18,60 +18,76 @@ param()
 #>
 
 Describe 'ErrorRecord' {
-    BeforeAll {
-        $ErrorRecord = try {
-            throw "A challenge to the sky!"
+    Context '$Error' {
+        BeforeAll{
+            <#
+                $Error is an automatic variable that contains a list of recent errors.
+                By calling $Error.Clear() we ensure past errors to not carry into our testing.
+            #>
+            $Error.Clear()
         }
-        catch {
-            $_
+        It 'sometimes has a reference to the object that caused the error' {
+            # But not always!
+            $Error[0].TargetObject | Should -BeNullOrEmpty
```

Comment:
If we can have this as part of a separate test case that throws an error with no target object (I think simple `throw` statements don't assign a target object, possibly?) this might be better served.

If not, it might be better to remove this assertion.

Reply:
I agree with just removing that assertion all together. Since we have basically that same assertion on line 38.